### PR TITLE
rewrite trento community checks regarding 'sbd' configuration

### PR DESCRIPTION
**Pull request by angelabriel**
fixing jsc#CFSA-1062, jsc#CFSA-1083, jsc#CFSA-1098, jsc#CFSA-1092

### CODE REVIEW

**Review by nelsonkopliku — CHANGES_REQUESTED** [review]
Very nice! Thanks!

I followed discussion https://github.com/trento-project/wanda/discussions/121
and left some comments about removing redundancy of some values conditions.

I also noticed an inconsistency in the values expectations for check `49591F` with the current implementation.

File: priv/catalog/0B6DB2.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+facts:
+  - name: sbd_SBD_PACEMAKER
+    #should only run on 'azure', 'nutanix' or 'kvm', NOT on 'gcp' or 'aws'
+    gatherer: sbd_config
+    argument: SBD_PACEMAKER
+
+values:
+  - name: expected_SBD_PACEMAKER
+    default: notUsed
+    conditions:
```
[the remark this answers is not in the document]
Comment (angelabriel) [reply priv/catalog/0B6DB2.yaml:60]: true. Fix is on it's way.

File: priv/catalog/49591F.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+facts:
+  - name: sbd_SBD_STARTMODE
+    # should only run on 'azure', 'nutanix' or 'kvm', NOT on 'gcp' or 'aws'
+    gatherer: CHANGE
+    argument: SBD_STARTMODE
+
+values:
+  - name: expected_SBD_STARTMODE
+    default: notUsed
+    conditions:
```
[the remark this answers is not in the document]
Comment (angelabriel) [reply priv/catalog/49591F.yaml:58]: It's the same as for the other sbd checks. It should only run on 'azure', 'kvm' and 'nutanix' and there the values needs to be 'always'. So a 'clean' will never be a valid value. The 'notUsed' is a leftover of my attempts to workaround the situation discussed in https://github.com/trento-project/wanda/discussions/121. I will fix it as all the other sbd checks.

File: priv/catalog/816815.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+facts:
+  - name: sbd_service_state
+    gatherer: systemd
+    argument: sbd
+
+values:
+  - name: expected_sbd_service_state
+    default: inactive
+    conditions:
```
[the remark this answers is not in the document]
Comment (angelabriel) [reply priv/catalog/816815.yaml:42]: fixed

**Review by nelsonkopliku — APPROVED** [review]

File: priv/catalog/0B6DB2.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-storage-protect.html
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-1.11.5.5.4.4
+
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-storage-protect.html
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-1.11.5.5.4.4
+
+facts:
+  - name: sbd_SBD_PACEMAKER
```
Comment (arbulu89) [inline priv/catalog/0B6DB2.yaml:52]: I would stick to small case `sbd_pacemaker`. Mixing looks strange, even though I know that you did it to use the "real" value

File: priv/catalog/49591F.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+facts:
+  - name: sbd_SBD_STARTMODE
+    gatherer: sbd_config
+    argument: SBD_STARTMODE
+
+values:
+  - name: expected_SBD_STARTMODE
+    # should only run on 'azure', 'nutanix' or 'kvm', NOT on 'gcp' or 'aws'
+    default: "clean"
```
Comment (arbulu89) [inline priv/catalog/49591F.yaml:57]: Let's remove quotes from here and next values. I see some checks with them and other without.
This is clear string, so it shouldn't be needed

File: priv/catalog/49591F.yaml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-1.11.5.5.4.4
+    - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-storage-protect.html
+
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-1.11.5.5.4.4
+    - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-storage-protect.html
+
+facts:
+  - name: sbd_SBD_STARTMODE
```
Comment (arbulu89) [inline priv/catalog/49591F.yaml:50]: The same for the small/capital letter convention

File: priv/catalog/61451E.yaml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    argument: SBD_DEVICE
+
+values:
+  - name: expected_multiple_SBD_DEVICE
+    # should only run on 'azure', 'nutanix' or 'kvm', NOT on 'gcp' or 'aws'
+    default: 3
+
+expectations:
+  - name: expectations_multiple_SBD_DEVICE
+    expect: facts.sbd_multiple_SBD_DEVICE.split(";").len() == values.expected_multiple_SBD_DEVICE
```
Comment (arbulu89) [inline priv/catalog/61451E.yaml:43]: Nice!

**Review by arbulu89 — COMMENTED** [review]
Super nice job!
I just did some nitpick comments about conventions of using `quotes` and small/capital letters.
Maybe we should discuss all of us to find the proper convention and write them down somewhere

**Pull request merged**